controller/work_item_link_type.go

```diff
@@ -238,12 +244,15 @@ func (c *WorkItemLinkTypeController) Show(ctx *app.ShowWorkItemLinkTypeContext)
 			linkCtx := newWorkItemLinkContext(ctx.Context, ctx.Service, appl, c.db, ctx.Request, ctx.ResponseWriter, hrefFunc, nil)
 			err = enrichLinkTypeSingle(linkCtx, &appLinkType)
 			if err != nil {
-				return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal("Failed to enrich link type: %s", err.Error()))
+				return goa.ErrInternal("Failed to enrich link type: %s", err.Error())
 			}
 			return ctx.OK(&appLinkType)
 		})
```

Comment:
comment on exported function ConvertWorkItemLinkTypeToModel should be of the form "ConvertWorkItemLinkTypeToModel ..."